app/src/main/java/org/kiwix/kiwixmobile/zim_manager/local_file_transfer/RequestEnableWifiP2pServicesDialog.java

```diff
@@ -0,0 +1,42 @@
+package org.kiwix.kiwixmobile.zim_manager.local_file_transfer;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.provider.Settings;
+import android.widget.Toast;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+import org.kiwix.kiwixmobile.R;
+
+import static org.kiwix.kiwixmobile.zim_manager.local_file_transfer.LocalFileTransferActivity.showToast;
+
+public class RequestEnableWifiP2pServicesDialog extends DialogFragment {
```

Comment:
Same question as RequestEnableLocationServicesDialog

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1276/commits/0a5d924b5f5ceddb3a67a0bad20397225c69d0e3#diff-e1d59ae6e15dcb502e125fb6e243d15dR507)

Reply:
Cool, as soon as you delete this class and the comments I will resolve

Reply:
ah, you have, didn't look at the latest code